### PR TITLE
React course modified

### PR DESCRIPTION
**Pull request by OmKanhe**
## What does this PR do?
Add resources

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -263,7 +263,9 @@
 * [React Tutorial](https://react-tutorial.app)
 * [React Tutorial](https://www.w3schools.com/react/) - W3Schools
 * [React Tutorial](https://www.scaler.com/topics/react/) - Scaler Topics
-
+* [React Tutorial](https://react-tutorial.app) - React Tutorial
```
Comment (Thenlie) [inline more/free-programming-interactive-tutorials-en.md:266]: This looks like a duplicate of line 263

**Review by Thenlie — CHANGES_REQUESTED** [review]
Please check the output of free-programming-books-lint and fix the errors. Update the title with `(changes made)` once the lint check is passing. Thanks!

File: more/free-programming-interactive-tutorials-en.md (hunk 1 above)
Comment (Thenlie) [reply more/free-programming-interactive-tutorials-en.md:266]: Maybe we could update this to credit the author, Jad Joubran.

**Comment by eshellman** [comment]
added in fe15c9c738c82b1d7c032585304ef63c7b6fa15e

**Pull request closed without merging**